engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -441,15 +466,6 @@ public Type instrumentor() {
     return instrumentor.getType();
   }
 
-  /**
-   * Returns the builtin module scope.
-   *
-   * @return the builtin module scope
-   */
-  public ModuleScope getScope() {
```

Comment:
- this removal is part of my thoughts on `ModuleScopeBuilder`
- basically a continuation of #13686
- `ModuleScope` is immutable state of a module _at a time_
- `Module` always points to a single `ModuleScope`, but it may change the scope it points to
    - as the user edits the _module source code_ new `ModuleScope` instances are created with every compilation
- e.g. `Module` is a **handle** - a persistent reference to an object representing the module in runtime
    - provides a name
    - but that's it
- `Builtins` need a persistent reference - e.g. they should only point to `Module`, not its scope
- even it doesn't make any difference in this case, as the scope of builtins never changes...